recipes/r-indicspecies/meta.yaml

```
@@ -15,10 +15,10 @@ source:
 build:
   merge_build_host: True  # [win]
   number: 0
-  noarch: generic
+  skip: True  # [win]
   rpaths:
     - lib/R/lib/
-    - lib/
+    - lib/  
```

No trailing whitespace:

```suggestion
    - lib/
```